src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -2521,7 +2525,8 @@ private void ApplyAutoEvoResults()
         // for reference in patch history (e.g. displaying it as zero on the species population chart)
         foreach (var entry in CurrentGame.GameWorld.Map.Patches)
         {
-            entry.Value.RecordConditions();
+            entry.Value.RecordSnapshot();
+            entry.Value.ClearLoggedEvents();
```

Comment:
Would it make sense to make ClearLoggedEvents private and make RecordSnapshot call it? I think it would be less dangerous to combine the recording of the events with clearing the events list, than leave this method public.

Reply:
I guess it would, I added a public clearing method in case the programmer wants an extra option to whether to clear the event records on snapshot recording or not though that should be possible by passing in a bool parameter nonetheless...

Reply:
Changed.